sections/futures/Trade/Trade.tsx

```diff
@@ -242,6 +242,8 @@ const Trade: React.FC<TradeProps> = () => {
 			<MarketInfoBox
 				availableMargin={futuresMarketsPosition?.remainingMargin ?? zeroBN}
 				buyingPower={sUSDBalance}
```

Comment:
I know this wasn't changed in this PR, but this value doesn't seem right to me. Shouldn't "buying power" refer to the amount you could invest given your deposited margin? I would think if "available margin" is 0 this should also be 0, but it shows your sUSD balance.